tests/npm-app/src/endpoints.js

```diff
@@ -0,0 +1,31 @@
+import * as _ from 'lodash-es'
+import protobuf from 'protobufjs/dist/protobuf.js'
+import * as rs  from 'jsrsasign';
+
+export function partition() {
+    // Example from https://lodash.com.
+    let arr = JSON.parse(body);
+    return JSON.stringify(_.partition(arr, n => n % 2));
+}
+
+export function pb() {
```

Comment:
Can we call this `protobuf` rather than `pb`? NPM's `pb` is a real and very different thing: https://npm.runkit.com/pb